server/src/main/kotlin/org/javacs/kt/Configuration.kt

```diff
@@ -57,6 +57,18 @@ data class InlayHintsConfiguration(
     var chainedHints: Boolean = false
 )
 
+data class KtFmtConfiguration(
+    var style: String = "google",
+    var indent: Int = 4,
+    var maxWidth: Int = 100,
+    var continuationIndent: Int = 8,
+    var removeUnusedImports: Boolean = true,
+)
+
+data class FormattingConfiguration(
+    var formatter: String = "ktfmt",
+    var ktFmt: KtFmtConfiguration = KtFmtConfiguration()
```

Comment:
Nit: The official name is `ktfmt`, so I wouldn't capitalize `Fmt`:

```suggestion
    var ktfmt: KtfmtConfiguration = KtfmtConfiguration()
```